python/sglang/srt/server_args.py

```diff
@@ -1210,6 +1211,15 @@ def add_cli_args(parser: argparse.ArgumentParser):
             "to record request metrics separately. This is especially useful when dp_attention is enabled, as "
             "otherwise all metrics appear to come from TP 0.",
         )
+        parser.add_argument(
+            "--tokenizer-metrics-custom-labels",
```

Comment:
Here, --tokenizer-metrics-custom-labels refers to the configuration of allowed labels, right? Could we rename it to a term that better aligns with its functional semantics, such as --tokenizer-metrics-allowed-labels?